README.md

```diff
@@ -23,19 +23,21 @@ npm install
 
 3. To expose your app server using an HTTP tunnel, install [ngrok](https://www.npmjs.com/package/ngrok#usage) globally, then start the ngrok service.
 
-    You can use `npm` to install ngrok:
+   You can use `npm` to install ngrok:
 
 ```shell
 npm install -g ngrok
 ```
 
-    Alternatively, MacOS users can use the [homebrew](https://brew.sh/) package manager:
+&#13;
+   Alternatively, MacOS users can use the [homebrew](https://brew.sh/) package manager:
```

Comment:
```suggestion
Alternatively, MacOS users can use the [homebrew](https://brew.sh/) package manager:
```

Reply:
If we remove these spaces we can probably delete the manual `&#13;` , as far as I can see in my IDE the rendered result is the same